src/betterproto/__init__.py

```diff
@@ -918,13 +995,41 @@ def from_dict(self: T, value: dict) -> T:
         return self
 
     def to_json(self, indent: Union[None, int, str] = None) -> str:
-        """Returns the encoded JSON representation of this message instance."""
+        """A helper function to parse the message instance into its JSON
+        representation.
+
+        This is equivalent to::
+
+            >>> return json.dumps(message.to_dict(), indent=indent)
+
+        Parameters
+        -----------
+        indent: Optional[Union[:class:`int`, :class:`str`]]
+            The indent to pass to :func:`json.dumps`.
+
+        Returns
+        --------
+        :class:`str`
+        """
         return json.dumps(self.to_dict(), indent=indent)
 
     def from_json(self: T, value: Union[str, bytes]) -> T:
-        """
-        Parse the key/value pairs in `value` into this message instance. This
-        returns the instance itself and is therefore assignable and chainable.
+        """A helper function to return the message instance from its JSON
+        representation. This returns the instance itself and is therefore assignable
+        and chainable.
+
+        This is equivalent to::
+
+            >>> return message.from_dict(json.loads(value))
```

Comment:
I don't think `>>>` is appropriate here since it suggests to me that we're talking about a REPL context.